### PR TITLE
python3Packages.mkdocs-drawio-exporter: init at 0.8.0

### PR DESCRIPTION
**Pull request by snpschaaf**
###### Description of changes

Add the mkdocs plugin mkdocs-drawio-exporter.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by snpschaaf** [comment]
@tfc

File: pkgs/development/python-modules/mkdocs-drawio-exporter/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  version = "0.8.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-b079XmbZIZvIiCInwv4S+lqOnLpzZVKZP69eYQWVahA=";
+  };
+
+  propagatedBuildInputs = [ mkdocs ];
```
Comment (tfc) [inline pkgs/development/python-modules/mkdocs-drawio-exporter/default.nix:19]: Reading the docs and source of this plugin i have seen that it's possible to define the drawio executable, but if you don't provide one they will search in the PATH environment: https://github.com/LukeCarrier/mkdocs-drawio-exporter/blob/master/mkdocsdrawioexporter/exporter.py#L192

In this case we can simply add `drawio-headless` to the propagated build inputs. This provides the user a working default config so they don't *have to* (but still *can* override via config) define a drawio executable path

Comment (snpschaaf) [reply pkgs/development/python-modules/mkdocs-drawio-exporter/default.nix]: Makes sense - adopted your suggestion with commit [6c831aa](https://github.com/NixOS/nixpkgs/pull/173730/commits/6c831aa8471072dd95a40bf7f74970f4609fd25f).

Comment (tfc) [reply pkgs/development/python-modules/mkdocs-drawio-exporter/default.nix]: Ok thank you. please squash that commit into your init-pkg commit and then it looks like good to go.

**Pull request merged**